.github/workflows/php.yml

```diff
@@ -32,16 +76,19 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
 
+      # Installing a specific version of Laravel will force us to compatible versions of `illuminate/http` and
+      # `illuminate/support`. If we didn't do this step then we'd always only ever test on the latest versions of those
+      # two packages.
       - name: Install dependencies
         run: |
           composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
-          composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
+          composer update --prefer-stable --prefer-dist --no-interaction --no-suggest
 
       - name: Check code standards
         run: composer run-script lint
 
-      - name: Execute static analysis
-        run: composer run-script psalm
+      # - name: Execute static analysis
+      #   run: composer run-script psalm
```

Comment:
Disabling Psalm from running in CI as we're having a lot of errors with it on PHP 8.1 running checks against vendor code when it shouldn't. Will revisit when Psalm v5 is out.